recipes/jupyter-archive/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyter-archive" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "c578affa23afcfebb1b2a7956db46f48dc6477cf804f8af8a1be638ffa039ac9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - python >=3.5
+    - pip
+    - jupyterlab >=1.0.0
+  run:
+    - python >=3.5
+    - jupyterlab >=1.0.0
+    - nodejs
```

Comment:
I know that for example Jupyterlab-git extension is defining nodejs as a dependency. But I personally don't like it. Because on infrastructure serving Jupyterlab as a service with preinstalled extensions but without extension manager for the end user, this install a unneeded dependency. 
But the counter argument, if the end user has not install node, it won't be able to build jlab is valid too. So up to you.

Reply:
Both arguments are valid IMO. I think I would prefer to let node for the moment. But we should ask a Jupyter dev if this should be removed or not. Let's worry about that once the recipe is accepted.

Reply:
Question asked: https://gitter.im/jupyterlab/jupyterlab?at=5da31edac87a1d28aca134dc